packages/gatsby-source-shopify/README.md

```diff
@@ -1,813 +1,282 @@
-| **Comment**        | A comment on a blog entry.                                                                                            |
-| **Collection**     | Represents a grouping of products that a shop owner can create to organize them or make their shops easier to browse. |
-| **Product**        | Represents an individual item for sale in a Shopify store.                                                            |
-| **ProductOption**  | Custom product property names.                                                                                        |
-| **ProductVariant** | Represents a different version of a product, such as differing sizes or differing colors.                             |
-| **ShopPolicy**     | Policy that a merchant has configured for their store, such as their refund or privacy policy.                        |
-| **ShopDetails**    | Name, description and money format that a merchant has configured for their store.                                    |
-
-For each data type listed above, `shopify${typeName}` and
-`allShopify${typeName}` is made available. Nodes that are closely related, such
-as `Article` and `Comment`, are provided as node fields as described below.
-
-**Note**: The following examples are not a complete reference to the available
-fields for each node. Utilize Gatsby's built-in GraphQL tool to discover the
-types and properties available.
-
-### Query articles
-
-The associated blog data is provided on the `blog` field. Article comments are
-provided on the `comments` field.
-
-```graphql
-{
-  allShopifyArticle {
-    edges {
-      node {
-        id
-        author {
-          email
-          name
-        }
-        blog {
-          title
-        }
-        comments {
-          id
-          author {
-            email
-            name
-          }
-          contentHtml
-        }
-        contentHtml
-        publishedAt(formatString: "ddd, MMMM Do, YYYY")
-      }
-    }
-  }
-}
-```
+**Note: Enabling Cart and Checkout features**
 
-### Query blogs
+If you are planning on managing your cart within Gatsby you will also need to check the box next to `Allow this app to access your storefront data using the Storefront API` and make sure to check `Read and modify checkouts`. This source plugin does not require Shopify Storefront API access to work, however, this is needed to add items to a Shopify checkout before passing the user to Shopify's managed checkout workflow. See [Gatsby Starter Shopify](https://github.com/gatsbyjs/gatsby-starter-shopify) for an example.
 
-Blog data is provided on the `blog` field on `Article`, but it can be queried
-directly like the following:
+Click the Save button and then click Create app to create your Private Shopify App. From there you can copy the API Key and Password from the Private app page and add them to your environment file for `SHOPIFY_ADMIN_API_KEY` and `SHOPIFY_ADMIN_PASSWORD` respectively.
+
+### Fire it up
```

Comment:
We don't normally have such information in a plugin README, I'd probably just remove this paragraph